index.html

```diff
@@ -138,6 +138,32 @@
         .bat-zero {
             color: #bd005e;
         }
+
+        .text-shiny {
+            position: relative;
+        }
+
+        .text-shiny::before {
+            content: "";
+            position: absolute;
+            top: 0;
+            left: 0;
```

Comment:
Can you try to either `margin: 0 2px;` or `padding` this please? There is no breathing room for the animation to shine.

Reply:
Achktually, simpler, and minor changes to gradient to make it fancier (in my humble opinion):

```css
/* remove position-relative shit above */

        .text-shiny {
            padding: 3px 4px;
            background-image: linear-gradient(
              45deg,
              goldenrod, palegoldenrod, goldenrod
            );
            background-size: 200% 100%;
            animation: shinyEffect 4s ease-in-out infinite;
            border-radius: 3px;
        }
```

Reply:
Ok hear me out:

```css
        .text-shiny {
           padding: 3px 4px;
           font-weight: bold;
           background-color:         goldenrod;
              background-image: linear-gradient(
              to right,
              transparent, rgba(238,232,170, 0.8), transparent
            );
            background-size: 200% 100%;
            animation: shinyEffect 3s linear infinite;
            border-radius: 3px;
            -webkit-background-clip: text;
            -webkit-text-fill-color: transparent;
        }

@keyframes shinyEffect {
            0% { background-position: -100% center; }
            50% { background-position: 100% center; }
            100% { background-position: -100% center; }
        }
```